### ISSUE TITLE
Skin needs to be compatible with newest version of USC.

### ISSUE THREAD

**Issue by Knifs** [nieknooijens/unnamed-sdvx-clone-sdvxIV-skin#1]
You need to update the skin so it works in USC otherwise it will not work.

**Comment by nieknooijens** [comment]
weird, it worked fine with https://github.com/Drewol/unnamed-sdvx-clone/commits/master
from 2 weeks ago, I did see some changes in textures in the last week so I guess I need to update some of them.

**Comment by nieknooijens** [comment]
with the version from yesterday it seems to be working just fine

**Comment by ttpcodes** [comment]
Running version `v0.3` of `USC` and it's crashing my game with the error that it could not find `titlescreen.lua`.

**Comment by rullinoiz** [comment]
yeah it kinda needs to be compatible because its missing a ton of scripts
